aten/src/ATen/xpu/detail/XPUHooks.cpp

```diff
@@ -30,6 +30,10 @@ int XPUHooks::getNumGPUs() const {
   return at::xpu::device_count();
 }
 
+void XPUHooks::deviceSynchronize(DeviceIndex device_index) const {
+  c10::xpu::syncStreamsOnDevice(device_index);
```

Comment:
pls give comments to tell the gap btw device sync and streams sync.

Reply:
done.